### PR TITLE
add "splash_offset"

### PR DESCRIPTION
**Pull request by MightyPlaza**
Adds "splash_offset"

closes: https://github.com/hyprwm/hyprpaper/issues/96

### CODE REVIEW

File: src/config/ConfigManager.cpp [hunk 1]
```diff
@@ -113,7 +113,11 @@ void CConfigManager::parseKeyword(const std::string& COMMAND, const std::string&
         g_pHyprpaper->m_bIPCEnabled = VALUE == "1" || VALUE == "yes" || VALUE == "on" || VALUE == "true";
     else if (COMMAND == "splash")
         g_pHyprpaper->m_bRenderSplash = VALUE == "1" || VALUE == "yes" || VALUE == "on" || VALUE == "true";
-    else
+    else if (COMMAND == "splash_offset") {
+        try {
+            g_pHyprpaper->m_iSplashOffset = std::stoi(VALUE);
+        } catch (std::exception& e) {}
```
Comment (vaxerski) [inline src/config/ConfigManager.cpp:119]: I'd log the error

**Comment by vaxerski** [comment]
also do we really want px or maybe %

**Comment by MightyPlaza** [comment]
done

**Review by vaxerski — APPROVED** [review]
splendid, thanks!

**Pull request merged**